File: Java/UberBlack.java
import java.util.Map;

public class UberBlack extends Car {
    
    Map<String, Map<String, Integer>> typeCarAccepted;
    Map<String, Integer> seatsMaterial;

    public UberBlack(String license, Account driver, Map<String, Map<String, Integer>> typeCarAccepted,
                     Map<String, Integer> seatsMaterial) {
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}

File: Python/UberBlack.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatsMAterial = []

    def __init__(self, license, driver, typeCarAccepted, seatsMaterial):
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMAterial = seatsMaterial